ci-operator/step-registry/gcp/deprovision/kms-key/gcp-deprovision-kms-key-commands.sh

```
@@ -0,0 +1,53 @@
+set -o pipefail
+
+if [ ! -f "${SHARED_DIR}/gcp_kms_key_ring" ]; then
+  echo "No KMS key ring file found in SHARED_DIR, nothing to clean up."
+  exit 0
+fi
+
+export CLOUDSDK_PYTHON=python3
+
+GOOGLE_PROJECT_ID="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
+export GCP_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/gce.json"
+sa_email=$(jq -r .client_email ${GCP_SHARED_CREDENTIALS_FILE})
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)